app/services/modalService.js

```diff
@@ -82,5 +102,26 @@ angular.module('portainer.services')
       callback: callback,
     });
   };
+
+  service.confirmContainerDeletion = function(title, text, callback) {
+    service.prompt({
+      title: title,
+      inputType: 'checkbox',
+      inputOptions: [
+        {
+          text: text,
+          value: '1'
```

Comment:
Could we use `1` as a value here instead of `'1'` ? This way we could remove the usage of the boolean `cleanAssociatedVolumes` and use the result directly (see comment above)